### PR TITLE
Feature/issue 35

### LINKED ISSUE #35: Update Pester3 task to automatically install latest 3.x version
The `Pester3` task should behave like the `Pester4` task and when no version is given, install the latest version of Pester 3.x. Make sure to not show any PackageManagement verbose import statements.

### PR DESCRIPTION
**Pull request by jlariono**
Fixes #35

### CODE REVIEW

File: Test/Invoke-WhiskeyPester3Task.Tests.ps1 [hunk 1]
```diff
@@ -283,6 +286,23 @@ Describe 'Invoke-WhiskeyPester3Task.when version of tool doesn''t exist' {
     Invoke-PesterTest -Path $pesterPassingPath -WithInvalidVersion -ShouldFailWithMessage $failureMessage -PassingCount 0 -FailureCount 0 -ErrorAction SilentlyContinue
 }
 
+Describe 'Invoke-WhiskeyPester4Task.when major version of tool is not 3.*' {
```
Comment (splatteredbits) [inline Test/Invoke-WhiskeyPester3Task.Tests.ps1]: Change `Pester4` to `Pester3`.

File: Test/Invoke-WhiskeyPester3Task.Tests.ps1 [hunk 2]
```diff
@@ -283,6 +286,23 @@ Describe 'Invoke-WhiskeyPester3Task.when version of tool doesn''t exist' {
...
 }
 
+Describe 'Invoke-WhiskeyPester4Task.when major version of tool is not 3.*' {
+    GivenTestContext
+    $version = '4.0.1'
+    $failureMessage = 'the major version number must always be ''3'''
+    Invoke-PesterTest -Path $pesterPassingPath -Version $version -ShouldFailWithMessage $failureMessage -PassingCount 0 -FailureCount 0 -ErrorAction SilentlyContinue
+}
+
+Describe 'Invoke-WhiskeyPester4Task.when Find-Module fails' {
```
Comment (splatteredbits) [inline Test/Invoke-WhiskeyPester3Task.Tests.ps1]: Change `Pester4` to `Pester3`.

File: Test/Invoke-WhiskeyPester3Task.Tests.ps1 (hunk 1 above)
Comment (jlariono) [inline Test/Invoke-WhiskeyPester3Task.Tests.ps1]: Fixed.

File: Test/Invoke-WhiskeyPester3Task.Tests.ps1 (hunk 2 above)
Comment (jlariono) [inline Test/Invoke-WhiskeyPester3Task.Tests.ps1]: Fixed.

**Pull request merged**